book/preliminary/checklist_index.md

```diff
@@ -42,8 +42,13 @@ Visit our [GitHub instruction page](checklist/github.md) to learn how!
 
 ### JupyterHub
 
-We will offer all tutorials within the Jupyter Hub computing environment.
-Visit our [Introduction to Jupyter Hub](checklist/jupyterhub.md) page to learn more!
+```{attention}
+If this is your first time connecting to the CryoCloud Jupyter Hub, please complete this short [application form](https://docs.google.com/forms/d/e/1FAIpQLScb2B5LRy7qU9qDfUp5a51n87sumOxivXbQhc02wFX_FxEbXg/viewform).
```

Comment:
We may want this note and link to the form on the jupyterhub page too (/preliminary/checklist/jupyterhub.md)